include/unifex/any_unique.hpp

```diff
@@ -433,9 +453,18 @@ class _byref<CPOs...>::type
     : vtable_(vtable_holder_t::template create<Concrete>())
     , impl_(std::addressof(impl)) {}
 
+  void swap(type& other) noexcept {
+    std::swap(vtable_, other.vtable_);
+    std::swap(impl_, other.impl_);
```

Comment:
I suppose you're not doing the two-step since you're not in a generic context and you know you want the `std` version?

Reply:
Yes.